README.md

```diff
@@ -89,6 +89,20 @@ Generating src/main/assembly/server-tarball.xml file from template src/main/asse
 Writing manifest state to .graylog-project-manifest-state
 ```
 
+### Initial Build
+
+Before you can run the server from the IDE, you have to run an initial build
+to create some assets.
+
+```
+$ cd graylog-project-repos/graylog2-server
+$ mvn compile
+$ cd ../graylog-plugin-pipeline-processor
```

Comment:
When this command is run I receive ``` cd: no such file or directory: ../graylog-plugin-pipeline-processor ``` because even though running the first compile build succeeds the path you are trying to point us to does not exist.